ramses/views.py

```diff
@@ -331,9 +331,12 @@ def update_many(self, **kwargs):
         by ES in the 'index' method (so user updates what he saw).
         """
         db_objects = self.get_dbcollection_with_es(**kwargs)
-        items = db_objects.all()
+
+        if not isinstance(db_objects, list):
```

Comment:
I think we don't need this check, `self.Model._update_many ` can handle list and query, so you can just pass db_objects to _update_many function